web-client/src/translations/en.json

```diff
@@ -138,7 +138,19 @@
       "contactUs": "Contact us",
       "logout": "Sign out",
       "switchToCAV": "Switch to Offering Support",
-      "switchToPIN": "Switch to Seeking Help"
+      "switchToPIN": "Switch to Seeking Help",
+      "settings": "Settings"
+    },
+    "settings": {
+      "heading": "Settings",
+      "changeName": "Change my name",
+      "deleteAccount": "Delete my account",
+      "changeNameForm": {
+        "full_name_error_message": "Please enter your name.",
```

Comment:
No punctuation at the end of template strings.

You will have to write in the code

```tsx
const foo = `${t('changeNameForm.full_error_message)}.`

Punctuation goes outside of the template string.  
```